boards/arm64/nxp_ls1046ardb/nxp_ls1046ardb_smp_defconfig

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Platform Configuration
+CONFIG_SOC_SERIES_LS1046A=y
+CONFIG_SOC_LS1046A=y
+CONFIG_BOARD_NXP_LS1046ARDB=y
+CONFIG_ARM64_VA_BITS_40=y
+CONFIG_ARM64_PA_BITS_40=y
+# 1.8GHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=1800000000
+
+# Zephyr Kernel Configuration
+CONFIG_SRAM_BASE_ADDRESS=0xc0000000
+CONFIG_SRAM_SIZE=1024
```

Comment:
same as before

Reply:
Removed, thanks.

Reply:
All done, thanks for yoru comments.

Jiafei.